pkg/recovery/recover_test.go

```diff
@@ -186,6 +186,7 @@ func TestFixUpBootstrapPods(t *testing.T) {
 					ReadOnly:  true,
 				}},
 			}},
+			HostNetwork: true,
```

Comment:
Should actually not set these in the `pods` array--the `fixUpBootstrapPods` method should add them.